charts/spire/charts/spire-server/values.yaml

```diff
@@ -390,6 +390,8 @@ upstreamAuthority:
     upstreamDriver: ""
     ## @extra upstreamAuthority.spire.server Server details for the Spire instance use as upstream CA
     server:
+      ## @param upstreamAuthority.spire.server.nameOverride Override the name for upstream Spire server
```

Comment:
Can we add a note on what the default will be if not overidden?

Reply:
It might make the rendered doc string rather long.

Its mostly used if your assembling a chart out of the subcharts and set nameOverride on the root spire server. then it should be set to the same nameOverride here too to have the default autocalculated value work.

Its used to align the names here:
* https://github.com/spiffe/helm-charts-hardened/blob/nested-chart/charts/spire-nested/values.yaml#L214
* https://github.com/spiffe/helm-charts-hardened/blob/nested-chart/charts/spire-nested/values.yaml#L123

For end users, either the default will just work, or they will set server.address instead.

Maybe we add a small bit of text to the doc string stating it should only be changed when building your own nested chart?

Reply:
Thats fine too

Reply:
kk. updated